core/lib/server/pub-capabilities.db.test.ts

```diff
@@ -2,12 +2,12 @@ import { describe, expect, it } from "vitest";
 
 import { CoreSchemaType, MemberRole } from "db/public";
 
-import type { Seed } from "~/prisma/seed/seedCommunity";
+import { createSeed } from "~/prisma/seed/createSeed";
```

Comment:
iirc importing `createSeed` up here did make data persist whereas importing just the `type { Seed }` didn't?

Reply:
yes, but i think that was bc they were both imported from `/seedCommunity`. I separated out the createSeed to a different file, which (I did not explicitly check) i think should fix that issue.
i do think using `createSeed` is slightly better as that preserves the type info once you pass it to `seedCommunity`, eg if you have `pubTypes: { 'SomepubType': ... } ` in your seed, the output of seedcommunity will have `pubTypes.SomePubType` as well